core-rust/state-manager/src/store/rocks_db.rs

```diff
@@ -133,12 +133,18 @@ enum RocksDBColumnFamily {
     /// Key: concat(account_address, state_version), value: null
     /// Given fast prefix iterator from RocksDB this emulates a Map<Account, Set<StateVersion>>
     AccountChangeStateVersions,
+    /// Additional details of "Scenarios" (and their transactions) executed as part of Genesis,
+    /// keyed by state version on top of which they are executed (note: this requires an assumption
```

Comment:
I'm not sure we need this note, or maybe we can say that we explicitly store Scenarios which committed transactions.